QuizApp/QuizApp/Mock/MockData.swift

```diff
@@ -0,0 +1,109 @@
+import UIKit
+
+public let sportColor = UIColor(red: 242 / 255, green: 201 / 255, blue: 76 / 255, alpha: 1)
```

Comment:
Please create a new file named Colors and add:
```
extension UIColor {
    static let sportCategory = UIColor(red: 242 / 255, green: 201 / 255, blue: 76 / 255, alpha: 1)
    ...
}
```

and use the constants in the app.